reason-mode.el

```diff
@@ -12,123 +12,17 @@
 ;; This file is distributed under the terms of both the MIT license and the
 ;; Apache License (version 2.0).
 
-;;; Commentary:
```

Comment:
Oh I spent a full 10 minutes to write this and now it goes away? Can we restore it?